label_studio/tasks/api.py

```diff
@@ -176,13 +180,15 @@ def update(self, request, *args, **kwargs):
     def get(self, request, *args, **kwargs):
         return super(AnnotationAPI, self).get(request, *args, **kwargs)
 
-    @swagger_auto_schema(auto_schema=None)
```

Comment:
same here, not sure this should be removed

Reply:
I turned it back